apps/web/next.config.ts

```diff
@@ -18,4 +19,14 @@ const nextConfig: NextConfig = {
   },
 };
 
-export default nextConfig;
+export default withSentryConfig(
+  withSentryConfig(nextConfig, {
+    org: "darklake",
+    project: "darklake",
+    silent: !process.env.CI,
+    widenClientFileUpload: true,
+    tunnelRoute: "/monitoring",
```

Comment:
Does this route exist, and is it set up to relay data to Sentry?

Some manual setup used to be required.

Reply:
in current versions, this should be automatically. Custom work is only needed if middleware interferes https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#step-6-avoid-ad-blockers-with-tunneling-optional